tutorials/fundamentals/trim.md

```diff
@@ -0,0 +1,26 @@
+To trim characters from a string in JavaScript, you must utilize the `replace()` function.
```

Comment:
To trim leading and trailing characters from a string in JavaScript, you should use the `replace()` function.